### PR TITLE
Added blogs app

### PR DESCRIPTION
**Pull request by jovanhyw**

### CODE REVIEW

File: apps/src/blogs/BlogsApp.vue [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    </div>
+
+    <div v-else>
+      <div class="row">
+        <div class="col">
+          <h3 class="has-text-weight-bold">Government Digital Services</h3>
+          <div class="row">
+            <div class="col is-4" v-for="blog in this.gdsBlogs.slice(0, 3)">
+              <a :href="blog.url" target="_blank">
+                <div class="sgds-card sgds-card-button" style="height: 100%;">
```
Comment (weijunyu) [inline apps/src/blogs/BlogsApp.vue]: Move inline styles to css classes if possible

File: apps/src/blogs/BlogsApp.vue [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                </div>
+              </a>
+            </div>
+          </div>
+          <p class="has-text-weight-semibold has-text-right is-size-8">
+            Read more at the
+            <a
+              href="https://blog.gds-gov.tech/"
+              target="_blank"
+            >Government Digital Services Blog</a>!
```
Comment (weijunyu) [inline apps/src/blogs/BlogsApp.vue]: Remove exclamation marks

File: apps/src/blogs/BlogsApp.vue (hunk 1 above)
Comment (jovanhyw) [reply apps/src/blogs/BlogsApp.vue]: fixed this in commit 00e89a8

File: apps/src/blogs/BlogsApp.vue (hunk 2 above)
Comment (jovanhyw) [reply apps/src/blogs/BlogsApp.vue]: fixed this in commit 00e89a8

**Pull request merged**